start of whats new page (#2349)
grommet/hpe-design-system: aries-site/src/pages/whats-new.mdx

<!-- BEGIN: What's New Entries -->

## January 24 - Febuary 4, 2022

### Communications
- **Office Hours** - A second office hours will be held on Thursdays, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact us if you would like an invite.

### Roadmap Updates 

> - **Page Layouts** 

Reviewer: ```suggestion
**Page Layouts** 
```

>  - Completed 2 column dashboard layout on [Design System site](https://design-system.hpe.design/templates/dashboards#two-column-dashboard).

Reviewer: ```suggestion
 - Completed [2 column dashboard layout](https://design-system.hpe.design/templates/dashboards#two-column-dashboard) on Design System site.
```

 - Started implementation on 3 column dashboard layout.

> - **No Data** 

Reviewer: ```suggestion
**No Data** 
```

<!-- BEGIN: What's New Entries -->

> ## January 24 - Febuary 4, 2022

Reviewer 2: Thinking we should extend this to cover last week's activities as well. If we are publishing, we should publish the most timely information.

Reviewer 2: I think we have all notable items covered from last week in the suggestions made below.

```suggestion
## January 24 - February 11, 2022
```

Author: sounds good

### Communications
- **Office Hours** - It was decided that the India office hours will be held every other Thursdays, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact us if you would like an invite.

### Roadmap Updates 
**Page Layouts** 
 - Completed [2 column dashboard layout](https://design-system.hpe.design/templates/dashboards#two-column-dashboard) on Design System site.
 - Started implementation on 3 column dashboard layout.

> **No Data** 

Reviewer 2: ```suggestion
- Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided [proof-of-concept project](https://github.hpe.com/grommet/hpe-global-header) to deliver the HPE Global Header as a Web Component.

**No Data** 
```

 - Added guidance as well as an example on how to display an empty value in a [NameValueList](https://design-system.hpe.design/components/namevaluelist#name-value-pair-with-empty-value).
**Navigation** 
 - Published all of the different type of navigations on the [Design System](https://design-system.hpe.design/templates/navigation) site.

### Stay current with the latest changes 

> - Added `position` prop to Toast to be aligned with [placement guidance](https://design-system.hpe.design/templates/toast-notifications#top-right-aligned-toast).

Reviewer 2: ```suggestion
- Enhanced ability to allow greater control over placement of Toast Notifications. Make sure you are aware of the trade-offs associated with [placement design decisions](https://design-system.hpe.design/templates/toast-notifications#placement).
```

> - Updated visual elements for each of the newly defined core principles of accessibility: perceivable, operable, understandable and robust; view the changes in our [accessibility page](https://design-system.hpe.design/foundation/accessibility).

Reviewer 2: ```suggestion
- Do you know the [4 core principles of accessibility](https://design-system.hpe.design/foundation/accessibility)? Read up on what it means to be perceivable, operable, understandable, and robust.
```

> - Added [Button ordering](https://design-system.hpe.design/components/button#button-ordering) based on their alignment.

Reviewer 2: ```suggestion
- Have "grouped buttons?" What types of buttons can be appear together? How should they be ordered? How should they be aligned? See [button ordering](https://design-system.hpe.design/components/button#button-ordering) for more detail.
```

> - Added a new Text color for `text-weak` 

Reviewer 2: We should be explaining what the change was and why it was made. Is this a breaking change? When will it be published to the HPE theme (i.e. published to NPM)?

Author: Ill go into detail in the next commit

> - Ideated on where to place a subtitle in a center Modal in [figma](https://www.figma.com/file/Ts53TAipMolmsv9DxWG3p0/HPE-Layer-Component?node-id=0%3A1). Example coming soon to Design System site.

Reviewer 2: Focus on completed work.

```suggestion
```

> 

Reviewer 2: ```suggestion
### Latest Releases
- [Grommet v2.20.1](https://github.com/grommet/grommet/releases/tag/v2.20.1) was released which includes an enhancement allowing form validation to occur "onMount" for controlled input forms, and bug fixes for "double focus on Select with search," DateInput formatting, Button icon alignment, and styling for TextInput with suggestions.
```

<!-- BEGIN: What's New Entries -->

## January 24 - February 11, 2022

> ### Hpe theme breaking change

Reviewer 2: ```suggestion
### HPE theme breaking change
```

> **Text-Weak** - Added a new Text color for `text-weak` per request there was a need for a new Gray text color. The new "weak" text is #757575 which passes accessibility for AA. It will be the new "weak" text, but will NOT be disabled. The previous text "weak" will now become the new text "xweak".

Reviewer 2: Is this change only for 'light-mode' values? Any affects to dark mode?

```suggestion
**Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575`. A new namespace `text-xweak` is being added with the value `#BBBBBB`.

The new `text-weak` value passed WCAG AA contrast values. `#BBBBBB` will remain the value for disabled fonts.
```

Author: no also dark mode Ill add that

>  We would like to hear any feedback on this change in order to test out this change before our next release. To test out this new change the directions can be found [here](https://github.com/grommet/grommet-theme-hpe#stable). 

Reviewer 2: ```suggestion
We would like to hear any feedback on this change ahead of the the next release of the HPE Theme. To test out this new change, apply the theme's stable branch to your project by [following these directions](https://github.com/grommet/grommet-theme-hpe#stable). 
```

The new `text-weak` value passed WCAG AA contrast values. `#BBBBBB` and `#606B7D` will remain the value for disabled fonts.

We would like to hear any feedback on this change ahead of the the next release of the HPE Theme. To test out this new change, apply the theme's stable branch to your project by [following these directions](https://github.com/grommet/grommet-theme-hpe#stable). 
 This is a breaking change that will be in the next HPE theme release which will be published in March. 

### Communications
- **Office Hours** - India office hours will be held every other Thursday, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact Kamlesh Thakur, if you would like a calendar invite.

### Roadmap Updates 

> **Page Layouts** 
>  - Published Content Layouts page with provides guidance for [designing for responsiveness](/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](/templates/content-layouts#breakpoints) and [page margins](/templates/content-layouts#page-margin).
>  - Began publishing a collection of [page layout templates](/templates/content-layouts#templates) and [dashboard templates](/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.
> - Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided proof-of-concept project to deliver the [HPE Global Header as a Web Component](https://github.hpe.com/grommet/hpe-global-header).

Reviewer 2: ```suggestion
- **Page Layouts**
  - Published Content Layouts page with provides guidance for [designing for responsiveness](/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](/templates/content-layouts#breakpoints) and [page margins](/templates/content-layouts#page-margin).
  - Began publishing a collection of [page layout templates](/templates/content-layouts#templates) and [dashboard templates](/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.
  - Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided proof-of-concept project to deliver the [HPE Global Header as a Web Component](https://github.hpe.com/grommet/hpe-global-header).
```

**Navigation** 
 - Published [five types of navigation patterns](https://design-system.hpe.design/templates/navigation#types-of-navigation), strengths of each and when to use.

### Stay current with the latest changes 
- Enhanced ability to allow greater control over placement of Toast Notifications. Make sure you are aware of the trade-offs associated with [placement design decisions](https://design-system.hpe.design/templates/toast-notifications#placement).
- Do you know the [4 core principles of accessibility](https://design-system.hpe.design/foundation/accessibility)? Read up on what it means to be perceivable, operable, understandable, and robust.
- Have "grouped buttons?" What types of buttons can be appear together? How should they be ordered? How should they be aligned? See [button ordering](https://design-system.hpe.design/components/button#button-ordering) for more detail.

> - Added a new Text color for `text-weak` per request there wsa aaa need for a new Gray text color. The new "weak" text is #757575 which passes accessibility for AA. It will be a "weak" text, but will NOT be disabled. The previous text "weak" will not become the new text "xweak". This is a breaking change that will be in the next HPE theme release which will be published in March. 

Reviewer 2: Is this necessary now that we have the placement above?

```suggestion
```

<!-- BEGIN: What's New Entries -->

## January 24 - February 11, 2022

### HPE theme breaking change

> **Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575` for light mode and from `606B7D` to `8C98AA` for dark mode. A new namespace `text-xweak` is being added with the value `#BBBBBB` for light mode and `#8C98AA` for dark mode.

Reviewer 2: #'s for consistency.

And, won't text-xweak receive the old dark color, since #8C98AA is the new one?

```suggestion
**Text-Weak** - To meet accessibility requirements, the color value for `text-weak` is changing from `#BBBBBB` to `#757575` for light mode and from `#606B7D` to `#8C98AA` for dark mode. A new namespace `text-xweak` is being added with the value `#BBBBBB` for light mode and `#606B7D` for dark mode.
```

> The new `text-weak` value passed WCAG AA contrast values. `#BBBBBB` will remain the value for disabled fonts.

Reviewer 2: ```suggestion
The new `text-weak` value passed WCAG AA contrast values. `#BBBBBB` and `#606B7D` will remain the value for disabled fonts.
```

> We would like to hear any feedback on this change ahead of the the next release of the HPE Theme. To test out this new change, apply the theme's stable branch to your project by [following these directions](https://github.com/grommet/grommet-theme-hpe#stable). 

Reviewer 2: Let's add a paragraph for visual separation.

```suggestion

We would like to hear any feedback on this change ahead of the the next release of the HPE Theme. To test out this new change, apply the theme's stable branch to your project by [following these directions](https://github.com/grommet/grommet-theme-hpe#stable). 
```

 This is a breaking change that will be in the next HPE theme release which will be published in March. 

### Communications
- **Office Hours** - India office hours will be held every other Thursday, 9:00 PM IST / 7:30 AM PST. This office hours is open to all. Contact Kamlesh Thakur, if you would like a calendar invite.

### Roadmap Updates 
**Page Layouts** 

>  - Published Content Layouts page with provides guidance for [designing for responsiveness] (https://design-system.hpe.design/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](https://design-system.hpe.design/templates/content-layouts#breakpoints) and [page margins](https://design-system.hpe.design/templates/content-layouts#page-margin).

Reviewer 2: Use NextJS SPA routing vs. page re-loads.

```suggestion
 - Published Content Layouts page with provides guidance for [designing for responsiveness](/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](/templates/content-layouts#breakpoints) and [page margins](/templates/content-layouts#page-margin).
```

> **Page Layouts** 
>  - Published Content Layouts page with provides guidance for [designing for responsiveness] (https://design-system.hpe.design/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](https://design-system.hpe.design/templates/content-layouts#breakpoints) and [page margins](https://design-system.hpe.design/templates/content-layouts#page-margin).

Reviewer 2: ```suggestion
- **Page Layouts** 
  - Published Content Layouts page with provides guidance for [designing for responsiveness] (/templates/content-layouts#designing-for-responsiveness), [responsive breakpoints](/templates/content-layouts#breakpoints) and [page margins](/templates/content-layouts#page-margin).
```

>  - Began publishing a collection of [page layout templates](https://design-system.hpe.design/templates/content-layouts#templates) and [dashboard templates](https://design-system.hpe.design/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.

Reviewer 2: ```suggestion
 - Began publishing a collection of [page layout templates](/templates/content-layouts#templates) and [dashboard templates](/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.
```

>  - Began publishing a collection of [page layout templates](https://design-system.hpe.design/templates/content-layouts#templates) and [dashboard templates](https://design-system.hpe.design/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.

Reviewer 2: Proper indentation of sub-bullets

```suggestion
  - Began publishing a collection of [page layout templates](https://design-system.hpe.design/templates/content-layouts#templates) and [dashboard templates](https://design-system.hpe.design/templates/dashboards#dashboard-templates) which provide customizable templates for design and development.
```

> - Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided [proof-of-concept project](https://github.hpe.com/grommet/hpe-global-header) to deliver the HPE Global Header as a Web Component.

Reviewer 2: Moving this to emphasize Global Header Web COmponent rather than "proof of concept"

```suggestion
- Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided proof-of-concept project to deliver the [HPE Global Header as a Web Component](https://github.hpe.com/grommet/hpe-global-header).
```

> - Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided [proof-of-concept project](https://github.hpe.com/grommet/hpe-global-header) to deliver the HPE Global Header as a Web Component.

Reviewer 2: ```suggestion
  - Collaborating with UCEP to align the Global Header/Footer Service with page layouts guidelines. Provided [proof-of-concept project](https://github.hpe.com/grommet/hpe-global-header) to deliver the HPE Global Header as a Web Component.
```

**No Data** 

>  - Added guidance and example [displaying empty/missing values](https://design-system.hpe.design/components/namevaluelist#name-value-pair-with-empty-value) in a NameValueList.

Reviewer 2: ```suggestion
 - Added guidance and example [displaying empty/missing values](/components/namevaluelist#name-value-pair-with-empty-value) in a NameValueList.
```

> **No Data** 
>  - Added guidance and example [displaying empty/missing values](https://design-system.hpe.design/components/namevaluelist#name-value-pair-with-empty-value) in a NameValueList.

Reviewer 2: ```suggestion
- **No Data** - Added guidance and example [displaying empty/missing values](/components/namevaluelist#name-value-pair-with-empty-value) in a NameValueList.
```

> **Navigation** 
>  - Published [five types of navigation patterns](https://design-system.hpe.design/templates/navigation#types-of-navigation), strengths of each and when to use.

Reviewer 2: ```suggestion
- **Navigation** - Published [five types of navigation patterns](/templates/navigation#types-of-navigation), the strengths for each, and when to use.
```

### Stay current with the latest changes 

> - Enhanced ability to allow greater control over placement of Toast Notifications. Make sure you are aware of the trade-offs associated with [placement design decisions](https://design-system.hpe.design/templates/toast-notifications#placement).

Reviewer 2: ```suggestion
- **Toast Notifications** - Enhanced ability to allow greater control over placement of Toast Notifications. Make sure you are aware of the trade-offs associated with [placement design decisions](/templates/toast-notifications#placement).
```

> - Do you know the [4 core principles of accessibility](https://design-system.hpe.design/foundation/accessibility)? Read up on what it means to be perceivable, operable, understandable, and robust.

Reviewer 2: ```suggestion
- **Accessibility** - Do you know the [4 core principles of accessibility](/foundation/accessibility)? Read up on what it means to be perceivable, operable, understandable, and robust.
```

> - Have "grouped buttons?" What types of buttons can be appear together? How should they be ordered? How should they be aligned? See [button ordering](https://design-system.hpe.design/components/button#button-ordering) for more detail.

Reviewer 2: ```suggestion
- **Buttons** - Have "grouped buttons?" What types of buttons can be appear together? How should they be ordered? How should they be aligned? See [button ordering](/components/button#button-ordering) for more detail.
```
